lib/src/widgets/evaluation_bar.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+const evaluationBarAspectRatio = 1 / 20;
+
+class EvaluationBar extends StatelessWidget {
```

Comment:
A doc comment would be welcome here. What it is? How is it different from the existing `EvaluationGauge`?

Reply:
It is for board thumbnail like there is in the website at the right of the board.

![20241123_16h06m38s_grim](https://github.com/user-attachments/assets/927146b1-8626-4eae-acb3-a99bd3aa90c7)

I don't know if I could reuse the `EvaluationGauge` for that instead.

Anyway I'll remove it from this PR as I should have it done it before.